pytest_html/plugin.py

```diff
@@ -296,7 +282,7 @@ def _appendrow(self, outcome, report):
             self.results.insert(index, result)
             tbody = html.tbody(
                 result.row_table,
-                class_="{0} results-table-row".format(result.outcome.lower()),
+                class_="{} results-table-row".format(result.outcome.lower()),
```

Comment:
we can call functions from inside the `f-string` what do you think about it?

```suggestion
                class_=f"{result.outcome.lower()} results-table-row",
```

Reply:
My personal preference is to use `f-string` for "pure" variables, and `format` for everything else (dicts, calling functions etc).

To me, the function call becomes a little bit obfuscated when "hidden away" in the f-string.

But if you feel strongly about it, I'm fine with having it that way. 😊